assignments/arrays.js

```diff
@@ -63,32 +74,109 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+const carInfo = id => {
+  let year = inventory[id - 1].car_year;
+  let make = inventory[id - 1].car_make;
+  let model = inventory[id - 1].car_model;
+  return `Car ${id} is a ${year} ${make} ${model}.`;
+};
+
+console.log(carInfo(33));
+console.log(carInfo(15));
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+
+const lastCarInfo = () => {
+  return `The last car in the inventory is a ${
+    inventory[inventory.length - 1].car_make
```

Comment:
Try pulling complex lines out into accessible variables, like so:
lastCar = inventory.length - 1;
console.log(`The last car is a ${inventory[lastCar].car_make} ${inventory[lastCar].car_model}`);

The code becomes more DRY and semantic.